### DISCUSSION TITLE
Form A4- Medications and RXNorm Ontology

### DISCUSSION THREAD

**Discussion by mllerch** [naccdata/uniform-data-set#59] [Forms]
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/mlerc/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<link rel=themeData
href="file:///C:/Users/mlerc/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/mlerc/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=EN-US style='tab-interval:.5in;word-wrap:break-word'>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>I
have several comments/questions about the UDS 4.0 A4 form<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>I
cannot find the following drugs in RxNorm however, they are listed on the A4
pdf:<o:p></o:p></span></p>

Altocor
--
Atamet
Budeprion
C Complex
Calciferol
Calcium Stool Softener
Carvedilol
Centrum Singles
Co Q-10
coenzyme R
Coreg
Dical-D
Dimetapp
Dioctyl SS
Disdol
Esidrix
Fempatch
FeroSul
Folic Acid
Hydrochloride
Hydrodiuril
Hytrin
Iron Supplement
K-Dur 10
K-Lor
Levothroid
Lopurin
Melatonin Time Release
Meloxicam
Nail-ex
Neuroforte-R,
Nico-400
Nicotenic Acid
Niravam
Nuprin
O-Cal-D
Omacor
Paxil CR
Reminyl
Replesta
Rolaids
Ryzolt
Senormin
Slow-K
Tempra
Toprol-XL
Urotrol
Vitamin C
Vitamin D3
Volmax

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>Iron
Supplement is not in RxNorm.&nbsp; Is that phrase being used in a general way?<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>It
seems that RxNorm does not have a separate listing for brand names when the
brand name is the exact same as the generic (FolicAcid, Meloxicam,
Carvedilol).&nbsp; The dropdown field is not case sensitive.<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
color:#211D1E'>The pdf has the wrong RxNorm code for fluticasone-salmeterol
(Advair).&nbsp; It should be 284635&nbsp; (not 41126 this belongs to plain
fluticasone).</span><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>“Rolaids
reformulated Aug 2006 “ (RxNorm 642335)&nbsp; is listed in RxNorm.&nbsp; I
would assume this is the code we would use for Rolaids in UDS 4.0.&nbsp; Is
that right?<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>Tempra
Quicklets (RxNorm 220207) is the only entry I can find in RxNorm.&nbsp; There
isn’t an entry for “Tempra”.&nbsp; Is this what you want to use for UDS 4.0?<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>Is
Paxil CR controlled release? &nbsp;RxNorm has “24 HR paroxetine hydrochloride
25 MG Extended Release Oral Tablet [Paxil]” (1738806).&nbsp; Is this what you
intended?<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>I
can’t find Nuprin in RxNorm.&nbsp; I can find Nuprin Backache (218815).&nbsp;
Is this what you intended?<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>Toprol-XL
is not in RxNorm.&nbsp; I can find metoprolol Extended Release Oral Tablet
[Toprol]&nbsp; (865577). &nbsp;Is this what you intended?<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>K-Lor
is not in RxNorm.&nbsp; I found Klor_Con (217905).&nbsp; Should I use this one?<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>O-Cal-D
is not in RxNorm.&nbsp; I found Os-Cal 500 with D&nbsp; (1243353).&nbsp; Is
this what you intended?<o:p></o:p></span></p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>

</body>

</html>

**Comment by SuzanneHunt-KC** [comment]
1/5/2024

I am documenting changes I made to the A4 form in REDCap so that it matches the pdf.

The form name on pdf in github has changed.  I changed the form name in REDCap  I also changed the survey name.  This really only changes the name that displays, not the name that is used behind the scenes.

The text UDS Coding Guidebook in the instructions should have a link embedded.  The URL is unknown at this time.

Instructions should be amended to indicate that dose and/or route are optional, not required.

The sentence in the instructions, about how to handle a medication not found, needs to change for REDCap.  I suspect that if a drug cannot be found in the drowdown, you won't find it in RxNav either.

I would suggest that the field label for anymeds be reworded to provide the intended temporality:
"Has the participant taken any (prescription?) medications in the last 2 weeks?"

Field names had not been released when I started building the A4 form, so I used the UDS3 naming convention, drugid.  Now I need to remove "id" in field names and in branching logic.  I downloaded the data dictionary as a csv and then uploaded the dictionary into REDCap.  I hope that didn't mess anything up for Laura and others.

**Comment by SuzanneHunt-KC** [comment]
1/5/2024  I suggest we add something to the REDCap instructions to mention RxNorm.  It is obvious on the paper form that RxNorm is to be used.  I think it is good practice to state it outright.

**Reply by mllerch** [reply]
@kgauthreaux for your review.
